js/main.js

```diff
@@ -28,17 +28,12 @@ var USERNAME = 'session/i@mir.lu';
 //definitions for coordinates
 var SCREEN_WIDTH = 360;
```

Comment:
Why do you have static values for the width of the screen?
Do all the smartwatches have 460 width?